Coffe/Features/DrinkList/DrinkListViewController.swift

```diff
@@ -5,15 +5,104 @@
 //  Created by macbook on 22.03.2025.
 //
 
-import UIKit
 import SwiftUI
+import UIKit
+import ComposableArchitecture
+import SnapKit
+
+@Reducer
+struct DrinkListItem {
```

Comment:
@LukasSkrivanek Added this to handle logic in cells, but since we are not doing it now we can maybe remove it later